OmiseSDK/Globals.swift

```diff
@@ -122,6 +122,10 @@ extension OMSSourceTypeValue {
             return "promptpay"
         case .payNow:
             return "paynow"
+        case .trueMoney:
+            return "truemoney"
+        case .pointsCiti:
+            return "points"
```

Comment:
Hmm, the `points` is deprecated from the Omise API. Could you change this to `points_citi`?

Reply:
That's just the prefix of the source. Will there be never any other `points` sourced added to the API later?

Reply:
I think there will be in the future. Anyway please this comment.